src/sap/fhir/model/r4/FHIRModel.js

```diff
@@ -64,6 +64,7 @@ sap.ui.define([
 	 * @param {boolean} [mParameters.x-csrf-token=false] The model handles the csrf token between the browser and the FHIR server
 	 * @param {object} [mParameters.filtering={}] The filtering options
 	 * @param {boolean} [mParameters.filtering.complex=false] The default filtering type. If <code>true</code>, all search parameters would be modelled via {@link https://www.hl7.org/fhir/search_filter.html _filter}
+	 * @param {boolean} [mParameters.securesearch=false] To enable RESTful search via {@link https://www.hl7.org/fhir/http.html#search POST}
```

Comment:
IMHO we should change the config object to `mParameters.search.secure`

Reply:
done